spec/unit/virtus/attribute/boolean/coerce_spec.rb

```diff
@@ -40,4 +40,20 @@
       end
     end
   end
+
+  context 'when nullify_blank is turned on' do
+    let(:options) { { :nullify_blank => true }}
+
+    context 'with a coercible input' do
+      let(:input) { 1 }
+
+      it { is_expected.to be(true) }
+    end
+
+    context 'with a blank input' do
+      let(:input) { '' }
+
+      it { is_expected.to be_nil }
+    end
+  end
 end
```

Comment:
:+1: - I'll move this specs to `spec/unit/virtus/attribute/coerce_spec.rb` since they aren't specific of the boolean attribute.